### ISSUE TITLE
PG_CONF parameters optimization

### ISSUE THREAD

**Issue by cayetanobv** [GeographicaGS/Docker-PostGIS#41]
We need to review PG_CONF parameters to optimize performance in a normal database operating context.

PG_CONF parameters are [here](https://github.com/GeographicaGS/Docker-PostGIS/blob/b54f6ff54f0a3200208daaf057d12de8c71c2e7d/Pleasant_Yacare/Dockerfile#L18)

**Comment by cayetanobv** [comment]
Logs parameters will be the first to change...

**Comment by alasarr** [comment]
I agree. I think it's a good starting point. I'll increase the checkpoint to 10min and set the log by default to ddl.

**Comment by josema-xyz** [comment]
Related: https://github.com/GeographicaGS/Docker-PostGIS/pull/42#issuecomment-379116629
